server/app/models/ProductionModel.ts

```diff
@@ -0,0 +1,41 @@
+import { Document, Schema } from 'mongoose';
+
+export interface IProduction extends Document {
+  type: string;
+  componentId: string;
```

Comment:
Is `componentId` useful to have on the request body schema?

I think it would be useful if on the frontend you are planning to allow users to select a pre-existing part/bike to produce more of, but I don't see that kind of usage for the `componentId` in the backend? Unless I missed it?

Reply:
@SiuYe What's your plan with this componentId? If it's not used then remove it

Reply:
@gordonpn thanks for the solution & help!